internal/planner/planner.go

```diff
@@ -24,12 +24,17 @@ type wasmBuiltin struct {
 
 // internalBuiltins are the built-in functions implemented in wasm.
 var internalBuiltins = map[string]wasmBuiltin{
-	ast.Plus.Name:     wasmBuiltin{ast.Plus, "opa_arithmetic_plus"},
-	ast.Minus.Name:    wasmBuiltin{ast.Minus, "opa_arithmetic_minus"},
-	ast.Multiply.Name: wasmBuiltin{ast.Multiply, "opa_arithmetic_multiply"},
-	ast.Divide.Name:   wasmBuiltin{ast.Divide, "opa_arithmetic_divide"},
-	ast.Abs.Name:      wasmBuiltin{ast.Abs, "opa_arithmetic_abs"},
-	ast.Rem.Name:      wasmBuiltin{ast.Rem, "opa_arithmetic_rem"},
+	ast.Plus.Name:         wasmBuiltin{ast.Plus, "opa_arithmetic_plus"},
+	ast.Minus.Name:        wasmBuiltin{ast.Minus, "opa_arithmetic_minus"},
+	ast.Multiply.Name:     wasmBuiltin{ast.Multiply, "opa_arithmetic_multiply"},
+	ast.Divide.Name:       wasmBuiltin{ast.Divide, "opa_arithmetic_divide"},
+	ast.Abs.Name:          wasmBuiltin{ast.Abs, "opa_arithmetic_abs"},
+	ast.Rem.Name:          wasmBuiltin{ast.Rem, "opa_arithmetic_rem"},
```

Comment:
Somewhat of a tangent.. but realized I must have misread/glossed over this in the previous reviews. Don't these strings need to match the actual c function definitions? I was looking through the code to refresh my memory on what we did for the `ast.Minus` builtin and sets and realized that `opa_arithmetic_minus` only shows up here, there is no other mention... do we map it to `opa_arith_minus` somewhere?

Reply:
It is peculiar, why the tests didn't fail. I fixed the function names to match.

Reply:
Currently the test.js skips without error reporting invalid wasm, hence the function name mismatch wasn't considered a test failure. It's something to improve in test framework. For now, I checked the added asset tests didn't trigger any skipping.

Reply:
I guess we did that to avoid failing on builtins back when we didn't support any of them, definitely something we can follow up on.